### ISSUE TITLE
CDA.pl support borked

### ISSUE THREAD

**Issue by Andrzej-Hochul** [yt-dlp/yt-dlp#7756]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I'm running yt-dlp version **2023.07.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Poland

### Provide a description that is worded well enough to be understood

I can't download the files, it used to work

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
D:\Backup\torrenty>yt-dlp -vU https://www.cda.pl/video/1050235423
[debug] Command-line config: ['-vU', 'https://www.cda.pl/video/1050235423']
[debug] Encodings: locale cp1250, fs utf-8, pref cp1250, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.19045-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: none
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.05.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[debug] Proxy map: {}
[debug] Loaded 1855 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
Current Build Hash: 5ff3e702171a50175c34397494e2d18ce35d771c2110b1e59bd173ec2fb352aa
yt-dlp is up to date (stable@2023.07.06)
[CDA] Extracting URL: https://www.cda.pl/video/1050235423
[CDA] 1050235423: Downloading webpage
WARNING: [CDA] unable to extract default player_json; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
ERROR: [CDA] 1050235423: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "yt_dlp\YoutubeDL.py", line 1560, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1709, in __extract_info
  File "yt_dlp\YoutubeDL.py", line 1768, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 2732, in process_video_result
  File "yt_dlp\YoutubeDL.py", line 1073, in raise_no_formats
yt_dlp.utils.ExtractorError: [CDA] 1050235423: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
```

**Comment by bashonly** [comment]
Is this an age-restricted video? Looks like same error as #5980

If so, this is a duplicate and there is a workaround:
> **selfisekai** commented on Apr 18
> as a workaround you can log in by username and password (not by cookies), which will make extractor use the mobile app API instead

or there is a PR that may fix this #6638

**Comment by Andrzej-Hochul** [comment]
Sorry not sure, i worked around it by viewing the source of the video from devlopers tools. and I used the not-logged in for reference. Many thanks for your time

**Comment by elvisef** [comment]
From what I see, the video url provided here is linking to the free to watch video.

From what I noticed, all DRM protected (i.e. premium) videos have "/vfilm" in the url. Even if there is no such a suffix, it is automatically linked while loading a DRM protected a.k.a. premium video.

I've tested the free video and it is available without a problem, working like a charm. I can watch it on my Firefox in Windows 7. Tested also in Incognito mode.

![firefox_2025-01-11_22-30-50](https://github.com/user-attachments/assets/974e07e8-2a7f-408c-be9c-34cf0362ca01)
![firefox_2025-01-11_22-28-06](https://github.com/user-attachments/assets/ccad7245-8859-484a-8e7b-02b44ab8965b)
